internal/future/serving/server.go

```diff
@@ -90,3 +129,78 @@ func (p *Params) AddHandleFunc(handlerFunc GrpcHandler, grpcProxyHandler GrpcPro
 	}
 	return p
```

Comment:
same here - why return p since p was already modified?

Reply:
For fluent syntax. See the new binaries in https://github.com/GoogleCloudPlatform/open-match/pull/336/files

Reply:
I am confused - new binaries use this as:
p.AddHandleFunc

so why are we returning p again??

Reply:
I removed it for now.